### ISSUE TITLE
the pressed property of the ToggleButton is not respected

### ISSUE THREAD

**Issue by simonseyock** [terrestris/react-geo#1710]
The pressed property of the ToggleButton is not respected. It works by setting the `selectedName` of the togglegroup.

See https://codesandbox.io/s/react-geo-pressed-ol38p?file=/index.js for an example

**Comment by simonseyock** [comment]
Maybe pressed could be calculated like this:
```
this.stat.selectedName === undefined ? child.props.pressed : this.state.selectedName === child.props.name
```
But then selectedName would need to get updated after and this would allow multiple buttons to be pressed at once.
The current behaviour is definitely easier, but if it should stay this way it would be nice if the documentation could be updated.

**Comment by Unraveler** [comment]
I agree on updating the Docs, it makes sense to have the group control the button that is pressed, but this is not at all clear
